src/coprocessor/dag/expr/builtin_encryption.rs

```diff
@@ -370,4 +533,79 @@ mod tests {
             assert_eq!(got, Datum::I64(exp));
         }
     }
+
+    const DATA: &[(&str, &str, &str, &str)] = &[
+        (
+            "aes-128-ecb",
+            "pingcap",
+            "1234567890123456",
+            "697BFE9B3F8C2F289DD82C88C7BC95C4",
+        ),
+        (
+            "aes-128-ecb",
+            "pingcap123",
+            "1234567890123456",
```

Comment:
Please also add tests for plain text len < 16 (for AES-128-xxx)

Reply:
done